tests/unit/utils/test_utils_record_getter.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+import mock
+
+from inspirehep.utils import record_getter
+
+
+class _MockPID(object):
+
+    def __init__(self, object_uuid):
+        self.object_uuid = object_uuid
+
+
+@mock.patch('inspirehep.utils.record_getter.PersistentIdentifier')
+@mock.patch('inspirehep.utils.record_getter.es')
+def test_get_es_record(es, pid):
+    pid.get.return_value = _MockPID('myuid')
+    with_es = {'ES': 'ES'}
+    es.get_source.return_value = with_es
+
+    # Test against hep/literature.
+    rec = record_getter.get_es_record('literature', 42)
+
+    assert with_es == rec
+    es.get_source.assert_called_with(
+        index='records-hep',
+        doc_type='hep',
+        id='myuid')
+
+
+@mock.patch('inspirehep.utils.record_getter.PersistentIdentifier')
+@mock.patch('inspirehep.utils.record_getter.Record')
+def test_get_db_record(record, pid):
+    pid.get.return_value = _MockPID('myuid')
+    with_db = {'DB': 'DB'}
+    record.get_record.return_value = with_db
+
+    # Test against hep/literature.
+    rec = record_getter.get_db_record('literature', 42)
+
+    assert with_db == rec
+    record.get_record.assert_called_with('myuid')
+
+
+def test_error_decorator():
+    @record_getter.raise_record_getter_error_and_log
+    def goodfn(record_type, recid):
+        return 42
+
+    assert goodfn(None, None) == 42
+
+    @record_getter.raise_record_getter_error_and_log
+    def badfn(record_type, recid):
+        raise Exception('message')
+
+    try:
+        badfn(None, None)
+    except record_getter.RecordGetterError as e:
+        pass
+    else:
+        assert False, 'Should raise RecordGetterError'
```

Comment:
You can use `pytest.raises` for this: https://pytest.org/latest/assert.html#assertions-about-expected-exceptions